bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -16,34 +16,33 @@
        specific language governing permissions and limitations
        under the License.
 */
-
-var Q = require('q');
-var fs = require('fs');
-var util = require('util');
-var path = require('path');
-var shell = require('shelljs');
-var spawn = require('cordova-common').superspawn.spawn;
-var events = require('cordova-common').events;
-var CordovaError = require('cordova-common').CordovaError;
-var check_reqs = require('../check_reqs');
-
-var GenericBuilder = require('./GenericBuilder');
-
-var MARKER = 'YOUR CHANGES WILL BE ERASED!';
-var SIGNING_PROPERTIES = '-signing.properties';
-var TEMPLATE =
+/* eslint no-self-assign: 0 */
+/* eslint no-unused-vars: 0 */
```

Comment:
I don't like introducing these comments, especially file global. Why do we need these?

Reply:
WTF?!